src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets

```diff
@@ -456,14 +455,16 @@ Copyright (c) .NET Foundation. All rights reserved.
   </Target>
 
   <Target Name="_GetDebugSymbolsWithTfm"
-          DependsOnTargets="DebugSymbolsProjectOutputGroup"
-          Condition="'$(IncludeBuildOutput)' == 'true'"
+          DependsOnTargets="DebugSymbolsProjectOutputGroup;$(TargetsForTfmSpecificDebugSymbolsInPackage)" 
           Returns="@(_TargetPathsToSymbolsWithTfm)">
-    <ItemGroup>
+    <ItemGroup Condition="'$(IncludeBuildOutput)' == 'true'">
       <_TargetPathsToSymbolsWithTfm Include="@(DebugSymbolsProjectOutputGroupOutput)">
         <TargetFramework>$(TargetFramework)</TargetFramework>
       </_TargetPathsToSymbolsWithTfm>
     </ItemGroup>
+    <ItemGroup>
+      <_TargetPathsToSymbolsWithTfm Include="@(TfmSpecificDebugSymbolsFile)" />
```

Comment:
Should this include the `NuGetRecursiveDir` metadata?  I couldn't quite follow from the source it that same path would be used for symbols or not: https://github.com/NuGet/NuGet.Client/blob/f24bad0668193ce21a1db8cabd1ce95ba509c7f0/src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTaskLogic.cs#L646

Reply:
cc @nkolev92

Reply:
I don't think that codepath would be used, but would be cool to check out whether scenarios like the one described in  https://github.com/NuGet/NuGet.Client/pull/2117.

Reply:
i tested it works. i can add a test for that as well.

Reply:
Yes, please!

Reply:
i pushed the test, can you trigger the ci ?

Reply:
Can you fix indent here? it looks whole xml tag off by1 space, of course it'll require another CI build.

Reply:
The indentation seems right 
<img width="940" alt="Screen Shot 2021-06-01 at 11 03 32 AM" src="https://user-images.githubusercontent.com/10516582/120370240-2fbc0b80-c2c9-11eb-8299-da778e1dfca5.png">

Reply:
Which test actually covers https://github.com/NuGet/NuGet.Client/pull/2117 work?

Reply:
PackCommand_ContentInnerTargetExtension_SymbolFilesDllWithRecursive covers some part of it. 
The automatic target path calculation wont work here because of this NuGet/Home#10860, hence we will to always specify the target path.